toolkit/partner-chains-cli/src/generate_keys/tests.rs

```diff
@@ -5,7 +5,7 @@ use scenarios::key_file_content;
 use scenarios::resources_file_content;
 
 const DATA_PATH: &str = "/path/to/data";
-const RESOURCES_CONFIG_PATH: &str = "partner-chains-cli-resources-config.json";
```

Comment:
Cosmetic: we could get rid of this and reference it from `config` crate in every place.